keeper/src/main/scala/zio/keeper/membership/SWIM.scala

```diff
@@ -327,7 +327,7 @@ final class SWIM(
     for {
       _       <- logger.info(s"sending $msg")
       payload <- msg.serialize
-      msg     <- serializeMessage(localMember_, payload, 1)
+      msg     <- serializeMessage("", localMember_, payload, 1)
```

Comment:
we should generate UUID here it's important to have tracing for internal messages.